client/state/receipts/types.ts

```diff
@@ -17,7 +17,7 @@ export interface ReceiptPurchase {
 	willAutoRenew: boolean;
 	saasRedirectUrl: string;
 	newQuantity: number | undefined;
-	blogId: number;
```

Comment:
Did you mean to do this? It's still set in the type below.

Reply:
`blogId` is still used [here](https://github.com/Automattic/wp-calypso/pull/79761/files#diff-0153186007a643acea15edec702a478f795b4219bc64762fbcacc8395cf84173R86) 👍

Reply:
That'll be the test failure :smile: :pray: